packages/eslint-plugin-pf-codemods/src/rules/helpers/nodeMatches/checkMatchingImportDeclaration.ts

```diff
@@ -0,0 +1,32 @@
+import { ImportDeclaration, ImportSpecifier } from "estree-jsx";
+import { pfPackageMatches } from "../pfPackageMatches";
+
+function findSpecifier(
+  node: ImportDeclaration,
+  imporSpecifier: ImportSpecifier
+) {
+  return node.specifiers.find(
+    (specifier) =>
+      specifier.type === "ImportSpecifier" &&
+      specifier.imported.name === imporSpecifier.imported.name
+  );
+}
+
+/** Used to check whether the current ImportDeclaration node matches at least 1 of the import specifiers. */
+export function checkMatchingImportDeclaration(
+  node: ImportDeclaration,
+  imports: ImportSpecifier | ImportSpecifier[],
+  exactMatch: boolean = false
+) {
+  if (!pfPackageMatches("@patternfly/react-core", node.source.value)) {
```

Comment:
Shouldn't the package here be customizable so that table or extensions or what have you are supported?

Reply:
Whoops